src/components/ScrollView/__stories__/Playground.stories.tsx

```diff
@@ -0,0 +1,95 @@
+import { ComponentStory } from '@storybook/react'
+import { ScrollView } from '../..'
+
+export const Playground: ComponentStory<typeof ScrollView> = ({
+  children,
+  ...props
+}) => (
+  <ScrollView style={{ height: '300px' }} {...props}>
+    {children}
+  </ScrollView>
+)
+
+Playground.args = {
+  children: `
+  Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vestibulum vel sem
```

Comment:
```suggestion
  children: `Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vestibulum vel sem
```

Reply:
This is done in another PR https://github.com/scaleway/scaleway-ui/pull/2098